recipes/libwebm/all/patches/1.0.0.29-cmake-dont-build-programs.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8db3881..501d757 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,19 +347,6 @@ add_library(webm ${libwebm_common_sources}
+             $<TARGET_OBJECTS:mkvmuxer>
+             $<TARGET_OBJECTS:mkvparser>)
+ 
+-add_executable(mkvparser_sample ${mkvparser_sample_sources})
```

Comment:
It could be pushed to the upstream as a new option 😄

Reply:
This was part of one of my patchsets, it has now been merged.